src/openrct2/actions/TrackPlaceAction.cpp

```diff
@@ -701,6 +649,55 @@ GameActions::Result TrackPlaceAction::Execute() const
         MapInvalidateTileFull(mapLoc);
     }
 
+    if (!(GetFlags() & GAME_COMMAND_FLAG_GHOST))
```

Comment:
Could you add a comment to this block something like "Update ride/track stats and interactions"